CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 - Chore: Enforce minimum quantity to 500 on dlc channel open
+- Chore(app): Remove paying lightning invoices
```

Comment:
```suggestion
- Chore(app): Disable paying lightning invoices
```

Reply:
The screen has been removed, so not just disabled :)